src/figuresClasses.ts

```diff
@@ -1,19 +1,103 @@
-export interface Figure {
 
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public sideA: number;
 
-}
+  public sideB: number;
+
+  public sideC: number;
```

Comment:
It is bad practice to make fields public when unnecessary, it violates the encapsulation OOP principle